SConstruct

```diff
@@ -164,6 +177,16 @@ def set_defaults(env, daos_version):
         #could refine this but for now, just assume these warnings are ok
         env.AppendIfSupported(CCFLAGS=PP_ONLY_FLAGS)
 
+    if GetOption("daos_ssd"):
+        patchfile = "/opt/crtdc/daos/master/patches/edv_build_latest.patch"
```

Comment:
If this patch isn't committed to the repository, what is the benefit of baking this into scons ?

And what is the benefit of doing it as a patch vs using ifdef's?  Seems it will get out of date very quickly

Reply:
It isn't straightforward as go doesn't support preprocessor macros so we can't use ifdef. There is probably a different way to do it with go code, but this is simpler solution for restrictive builds for edv cluster until a generalized solution is developed.